docs/UserGuide.md

```diff
@@ -228,19 +228,25 @@ Use this command to edit his/her details easily!
 
 **Syntax:**
 
-`edit INDEX [{SPECIFIER}/{DATA}]`
+`edit INDEX {SPECIFIER}/{DATA}...`
```

Comment:
This syntax could use some explaining in the `Notes about the command format` section

Reply:
I think this already has a point about it in the referred to section